docs/examples/combined/newsletter-summarizer-javascript.mdx

```diff
@@ -154,11 +154,11 @@ description: "This project serves as an example of how to use Composio to seamle
     // Function to set up the GitHub connection for the user if it doesn't exist
     async function setupUserConnectionIfNotExists(toolset, entityId, c) {
         const entity = await toolset.client.getEntity(entityId);
-        const connection = await entity.getConnection('github');
+        const connection = await entity.getConnection({ appName: "GITHUB" });
 
         if (!connection) {
             // If the user hasn't connected their GitHub account
-            const connection = await entity.initiateConnection('github');
+            const connection = await entity.initiateConnection({appName: "github"});
```

Comment:
@abhishekpatil4 Can you run prettier over the files? It's not formatted correctly now.

Reply:
In some places, it's "GITHUB," and here, it's "github" (casing). Let's use consistent casing.

Reply:
Also, if it's not case-sensitive, add a comment to the docs or code.

Reply:
Sure!